src/components/DataTable/DataTable.tsx

```diff
@@ -126,6 +129,17 @@ const DataTablePresentationComponent = memo(
             [setStore],
         );
 
+        const viewabilityConfigCallbackPairs = useMemo<ViewabilityConfigCallbackPairs>(
+            () => [
+                {
+                    viewabilityConfig: DEFAULT_VIEWABILITY_CONFIG,
```

Comment:
if DEFAULT_VIEWABILITY_CONFIG is coming from a separate file, onViewableItemsChanged should also come from a the same file.

I would abstract it into a hook that gives you a ViewabilityConfigCallbackPair